console/module/device/src/main/java/org/eclipse/kapua/app/console/module/device/client/device/tag/DeviceTabTags.java

```diff
@@ -29,6 +29,7 @@
     public DeviceTabTags(GwtSession currentSession) {
         super(currentSession, DEVICES_MSGS.tabTagsTitle(), new KapuaIcon(IconSet.TAGS));
         deviceTagGrid = new DeviceTagGrid(null, currentSession, selectedEntity);
+        setEnabled(false);
```

Comment:
What I'm thinking for the Events tab also stands true for this one. I wouldn't disable it.